matchbox_socket/src/webrtc_socket/wasm.rs

```diff
@@ -299,7 +299,7 @@ impl Messenger for WasmMessenger {
 async fn complete_handshake(
     signal_peer: SignalPeer,
     conn: RtcPeerConnection,
-    received_candidates: Vec<PeerId>,
+    received_candidates: Vec<String>,
```

Comment:
Why was this changed to String?

Reply:
This was actually a bug in the existing code. Candidates are the string payload in the IceCandidate event. Just got exposed now that PeerId is no longer an alias.